<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="search_btn">
      <property name="text">
       <string>얼굴조사하기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="init_btn">
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>317</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="person_info">
      <property name="font">
       <font>
        <family>나눔고딕 ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>인물 정보</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="sex">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>성별</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sex_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="sex_accuracy">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>성별 정확도</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sex_accuracy_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="age">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>예상 나이</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="age_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="age_accuracy">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>나이 정확도</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="age_accuracy_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="emotion">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>기분</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="emotion_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="emotion_accuracy">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>기분 정확도</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="emotion_accuracy_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="pose">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>얼굴 방향</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pose_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="pose_accuracy">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>얼굴 방향 정확도</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pose_accuracy_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>480</y>
     <width>321</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="person_celebrity">
      <property name="font">
       <font>
        <family>나눔고딕 ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>닮은 연예인</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="celebrity_num">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>닮은 연예인 수</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="celebrity_num_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="celebrity_name1">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>닮은 연예인1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="celebrity_name1_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="celebrity_accuracy1">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>정확도</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="celebrity_accuracy1_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="celebrity_name2">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>닮은 연예인2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="celebrity_name2_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="celebrity_accuracy2">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>정확도</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="celebrity_accuracy2_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="celebrity_name3">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>닮은 연예인3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="celebrity_name3_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLabel" name="celebrity_accuracy3">
        <property name="font">
         <font>
          <family>나눔고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>정확도</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="celebrity_accuracy3_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="photoView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>400</width>
     <height>700</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
